hail/hail/src/is/hail/io/plink/ExportPlink.scala

```diff
@@ -49,14 +49,14 @@ class BitPacker(nBitsPerItem: Int, os: OutputStream) extends Serializable {
 
   private def write(): Unit =
     while (nBitsStaged >= 8) {
-      os.write(data.toByte)
+      os.write(data.toInt)
```

Comment:
This is relying very heavily on the contract that `OutputStream#write(int)` only writes the low byte of this int. I think that's safe, as the contract for `write` across the ecosystem is very strong, but I wanted to make sure that we're all aware of this.